### PR TITLE
Added new docker image info in README

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[Added new docker image info in README]
### Issues Resolved
[]
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
- [x] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,7 +127,20 @@ All you need is Docker and git (optional).
 
    Note: You can navigate to the repo url and download the repo as a zip file if you don't want to use git
 
-2. Build the docker image: `$ docker build -t puppet-oneview .`
+2. You can build docker image locally or pull the docker image from Docker Hub.
+   * Build the docker image locally: `$ docker build -t puppet-oneview .`
```
Comment (chebroluharika) [inline README.md:132]: Mention how to run the image locally also.

File: README.md [hunk 2]
```diff
@@ -157,6 +170,9 @@ All you need is Docker and git (optional).
      -e IMAGE_STREAMER_SSL_ENABLED=true \
      puppet-oneview puppet apply image_streamer/deployment_plan.pp --debug --trace
    ```
+
+#### Using Docker Hub Image
```
Comment (chebroluharika) [inline README.md]: It should be near pull docker image from Docker Hub

File: README.md (hunk 1 above)
Comment (nabhajit-ray) [reply README.md:132]: Running of image is common for both local image and docker hub image  and it is part of section 3

File: README.md (hunk 2 above)
Comment (nabhajit-ray) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (chebroluharika) [reply README.md:132]: way of running would be same but not image names

File: README.md [hunk 3]
```diff
@@ -157,6 +175,7 @@ All you need is Docker and git (optional).
      -e IMAGE_STREAMER_SSL_ENABLED=true \
      puppet-oneview puppet apply image_streamer/deployment_plan.pp --debug --trace
    ```
+
```
Comment (chebroluharika) [inline README.md:183]: Mention these run steps below build docker images

File: README.md [hunk 4]
```diff
@@ -127,7 +127,32 @@ All you need is Docker and git (optional).
...
+     -e ONEVIEW_USER='Administrator' \
+     -e ONEVIEW_PASSWORD='secret123' \
+     -e ONEVIEW_SSL_ENABLED=true, \
+     -e ONEVIEW_LOG_LEVEL='info' \
+     -e ONEVIEW_API_VERSION=800 \
+     hewlettpackardenterprise/hpe-oneview-sdk-for-puppet:v2.6.0-OV5.2 puppet apply fc_network.pp --debug --trace
+   ```
+
 
 3. Now you can run any of the example manifests in this directory:
```
Comment (chebroluharika) [inline README.md]: Remove 3 and please mention this as "Now you can run any of the example manifests in this directory locally"

**Review by chebroluharika — APPROVED** [review]

**Review by yuvirani — APPROVED** [review]

**Pull request merged**